resources/migrations/001_update_migrations.yaml

```
@@ -9383,6 +9383,242 @@ databaseChangeLog:
+                  type: ${timestamp_type}
+                  remarks: The timestamp of when the handler was updated
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: v51.2024-09-05T08:00:05
+      author: qnkhuat
+      comment: index on notification_handler.notification_id
+      rollback: # not necessary, will be removed with the table
+      preConditions:
+        - not:
+            - indexExists:
+                tableName: notification_handler
+                indexName: idx_notification_handler_notification_id
+      changes:
+        - createIndex:
+            tableName: notification_handler
+            columns:
+              - column:
+                  name: notification_id
+            indexName: idx_notification_handler_notification_id
+
+  - changeSet:
+      id: v51.2024-09-05T08:00:06
+      author: qnkhuat
+      comment: index on notification_handler.channel_id
+      rollback: # not necessary, will be removed with the table
+      preConditions:
+        - not:
+            - indexExists:
+                tableName: notification_handler
+                indexName: idx_notification_handler_channel_id
+      changes:
+        - createIndex:
+            tableName: notification_handler
+            columns:
+              - column:
+                  name: channel_id
+            indexName: idx_notification_handler_channel_id
+
+  - changeSet:
+      id: v51.2024-09-05T08:00:07
+      author: qnkhuat
+      comment: Create the notification_recipient table
+      preConditions:
+        - not:
+          - tableExists:
+              tableName: notification_recipient
+      changes:
+        - createTable:
+            tableName: notification_recipient
+            remarks: who should receive the notification
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: notification_handler_id
+                  type: int
+                  remarks: the handler that the recipient is connected to
+                  constraints:
+                    nullable: false
+                    referencedTableName: notification_handler
+                    referencedColumnNames: id
+                    foreignKeyName: fk_notification_recipient_notification_handler_id
+                    deleteCascade: true
+              - column:
+                  name: type
+                  type: varchar(64)
+                  remarks: the type of the recipient
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: int
+                  remarks: an user if the recipient has type user
```

[English is weird](https://english.stackexchange.com/a/105117) (article is based on sound, not spelling)

```suggestion
                  remarks: a user if the recipient has type user
```

---

I kept forgetting this!